Controllers/auth/index.js

```diff
@@ -0,0 +1,15 @@
+const {
+  login,
```

Comment:
I don't think this will work, shouldn't it be like: 
```js
const login = require("./login");
module.exports = {...login}
```

Reply:
i am exporting the handler function itself like `module.exports = login`...so i guess there should be no problem

Reply:
you are using 
```js
require("./");
```
which is requiring the index file itself, how will that code work?